### DISCUSSION TITLE
Supressions - manual remove

### DISCUSSION THREAD

**Discussion by kluvi** [postalserver/postal#2200] [Feature suggestions]
There is no button to manualy remove recipient from supressions list. I must go to database, find right database name and manualy remove record from table.

**Comment by willpower232** [comment]
You should be able to manually release a held message sent to a suppressed recipient and if that is delivered successfully, they will be removed from the list.

**Reply by kluvi** [reply]
Thank you. But I think, that it will be more straightforward, if there will be some button directly in supressions list. It can then release all held messages for this address at once.

**Comment by dietmarscharf** [comment]
I also need that functionality to remove an address from the suppresion list manually with a simple button click. I would soon start a PR for this if this is fine for @willpower232?

**Reply by willpower232** [reply]
I don't have any say on the code that gets shipped, if you prepare the code though the worse case scenario is that you have to make your own docker image

**Reply by sinnbeck** [reply]
@dietmarscharf Did you get this PR done? :) I am hoping for the same feature in the future

**Reply by ugurtam** [reply]
It would be very nice, any actions yet ?

**Reply by ghost** [reply]
I think you can just delete them from the table, but someone more familiar with the software can confirm.

mariadb postal-server-1
select * from suppressions;
delete from suppressions where id=1;

Disclaimer: I just did it and it worked, but I don't know if I missed anything. Also, not saying there shouldn't be a feature.
